t/t-large-files-on-windows.sh

```diff
@@ -5,6 +5,7 @@ test_description='test large file handling on windows'
 
 test_expect_success SIZE_T_IS_64BIT 'require 64bit size_t' '
 
+	test-tool zlib-compile-flags >zlibFlags.txt &&
```

Comment:
While this might have helped developing this patch series, I doubt that it will help anything in the future.

The commit message talks about *what* this patch does, but not *why*. I'd rather have it talk about the "why?" than what I can readily read in the patch...

Reply:
Yes. This is an incomplete test pre-requiste. We actually need to check the value exported by  the zlib-compile-flags and see if it was compiled with a 4byte long.

Should also check the gnu crc32 compile settings (the length of the crc sequence is set as 'long', so again 4byte limit)

Reply:
I have a growing feeling that this is going in the exact wrong direction.

It would appear that this direction leads us to a regression test that verifies that zlib is compiled with unfortunate settings.

When what we really want to test for is that the regression is fixed where we can handle pack files larger than 4GB on systems whose `size_t` is 64-bit.

I guess I got lured in that direction when I suggested to put more prerequisites than just `SIZE_T_IS_64BIT`, and I was just as wrong: if this test case passes on a system where `size_t` *and* `unsigned long` are 64-bit, who cares?

Reply:
> When what we really want to test for is that the regression is fixed where we can handle pack files larger than 4GB on systems whose `size_t` is 64-bit.

.... No matter what the bit size of `long` is.

Reply:
Surely it's the _library_ that matters, not what we compiled our code with. i.e. that we may have inappropriate linkage between two different expectations. 

zlib itself happily works on both 32 and 64 bit pointer systems, it just can't cope with data chunks longer than the size of long that compiled the library in some dim distant past. At least that's the part I took from their FAQs, and why they provided that convenience function for user code to check.

I am cautious that we don't over trim the tests such that all the issues that prevented this being fixed 'years ago' could creep back out of the woodwork.